components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/org/wso2/carbon/identity/application/mgt/ui/i18n/Resources.properties

```diff
@@ -115,6 +119,8 @@ application.list.error.while.removing.app=Error while removing application
 application.passive.sts.realm=Passive STS Realm
 application.passive.sts.wreply=Passive STS WReply URL
 application.openid.realm= OpenID Realm
+application.successfull=Application registered successfully
+application.regenerated=Client Secret successfully updated for Client ID:
```

Comment:
Better to parameterize this like "...... for Client ID: {0}" and update the code. The reason is, some languages may not be able to have this value at the end of the sentence.